commands/release.go

```diff
@@ -195,6 +202,45 @@ func showRelease(cmd *Command, args *Args) {
 	os.Exit(0)
 }
 
+func downloadRelease(cmd *Command, args *Args) {
+	tagName := cmd.Arg(0)
+	if tagName == "" {
+		utils.Check(fmt.Errorf("Missing argument TAG"))
+	}
+
+	localRepo, err := github.LocalRepo()
+	utils.Check(err)
+
+	project, err := localRepo.MainProject()
+	utils.Check(err)
+
+	gh := github.NewClient(project.Host)
+
+	release, err := gh.FetchRelease(project, tagName)
+	utils.Check(err)
+
+	for _, asset := range release.Assets {
+		assetReader, apiError := gh.DownloadReleaseAsset(asset.ApiUrl)
+		utils.Check(apiError)
+		if assetReader == nil {
```

Comment:
Oh, good catch. I saw this pattern somewhere else.

> When err is nil, resp always contains a non-nil resp.Body. Caller should close resp.Body when done reading from it.

https://golang.org/pkg/net/http/#Client.Do